packages/react-sdk/src/component-generator.test.tsx

```diff
@@ -948,3 +948,64 @@ test("generate resource prop", () => {
     "
   `);
 });
+
+test("skip unsafe properties", () => {
+  expect(
+    generateWebstudioComponent({
+      classesMap: new Map(),
+      scope: createScope(),
+      name: "Page",
+      rootInstanceId: "body",
+      parameters: [],
+      dataSources: new Map(),
+      indexesWithinAncestors: new Map(),
+
+      instances: new Map([
+        [
+          "body",
+          { type: "instance", id: "body", component: "Body", children: [] },
+        ],
+      ]),
+      props: new Map([
```

Comment:
you can use toMap utility with array of instances or array of props, for readability.